### ISSUE TITLE
Missing alt text, link descriptions

### ISSUE THREAD

**Issue by rediaz111** [fraser-lab/fraser-lab.github.io#43]
Certain images are missing alt text, like the Beer and Tacos acrostic on `/compact`. Links do not contain descriptive text or indicate if the link will open in a new window or tab.
